plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -183,18 +193,18 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 			};
 		},
 
-		_handleCodeMirrorChange: function () {
-			var newData = this.codeMirrorEditor.getValue();
-			var preview = this.dialog
-				.getContentElement('main', 'preview')
-				.getElement();
-
-			var iframe = preview.findOne('iframe');
-			if (iframe && iframe.$) {
-				var iframeDocument = iframe.$.contentDocument;
-				var iframeBody = iframeDocument.body;
-				iframeBody.innerHTML = newData;
-			}
+		_sanitizeHTML: function (html) {
+			var sanitizedHtml = html
+				.replace(HTML_TAG_WITH_ON_ATTRIBUTE_REGEX, function (match) {
+					return match.replace(ON_ATTRIBUTE_REGEX, '');
+				})
+				.replace(ALERT_REGEX, '')
+				.replace(INNER_HTML_REGEX, '')
+				.replace(PHP_CODE_REGEX, '')
+				.replace(ASP_CODE_REGEX, '')
+				.replace(ASP_NET_CODE_REGEX, '');
+
+			return sanitizedHtml;
```

Comment:
@fortunatomaldonado Are we intentionally removing `_handleCodeMirrorChange`? It seems to update preview iframe. Won't this change result in preview never updating?